e2e/Pages/DashboardPage.ts

```diff
@@ -0,0 +1,56 @@
+import { Locator, Page } from 'playwright';
+
+class DashboardPage {
+  readonly page: Page;
+  readonly floodIcon: Locator;
+  readonly heavyRainIcon: Locator;
+  readonly droughtIcon: Locator;
+  readonly dashboardDevControlButton: Locator;
+  readonly countrySwitcherDropdown: Locator;
+  readonly countrySwitcherDropdownOption: Locator;
+  readonly dashboardDevControlCloseButton: Locator;
+  readonly loader: Locator;
+
+  constructor(page: Page) {
+    this.page = page;
+    this.floodIcon = this.page.getByTestId('disaster-type-button-floods');
+    this.heavyRainIcon = this.page.getByTestId(
+      'disaster-type-button-heavy-rain',
+    );
+    this.droughtIcon = this.page.getByTestId('disaster-type-button-drought');
+    this.dashboardDevControlButton = this.page.getByTestId(
+      'dashboard-dev-control-button',
+    );
+    this.countrySwitcherDropdown = this.page.getByTestId(
+      'country-switcher-dropdown',
+    );
+    this.countrySwitcherDropdownOption = this.page.locator('ion-radio');
+    this.dashboardDevControlCloseButton = this.page.getByTestId(
+      'dashboard-dev-control-close-button',
+    );
+    this.loader = this.page.getByTestId('loader');
+  }
+
+  async navigateToFloodDisasterType() {
+    await this.floodIcon.click();
+  }
+
+  async navigateToHeavyRainDisasterType() {
+    await this.heavyRainIcon.click();
+  }
+
+  async navigateToDroughtDisasterType() {
+    await this.droughtIcon.click();
+  }
+
+  async switchToCountryByName({ countryName }: { countryName: string }) {
```

Comment:
Use the login for the specific country instead of using the dev tools for this navigation. Here is the [Uganda user](https://github.com/rodekruis/IBF-system/blob/5bb38d45f41e84c4e890b04315f7593b8bdb8be7/services/API-service/src/scripts/json/users.json#L11-L20).

Reply:
Removed the function and replaced with with Uganda user login.